code/modules/surgery/organs/organ_external.dm

```diff
@@ -552,6 +552,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 				"<span class='danger'>\The [owner]'s [name] explodes[gore]!</span>",\
 				"<span class='moderate'><b>Your [name] explodes[gore]!</b></span>",\
 				"<span class='danger'>You hear the [gore_sound].</span>")
+			disembowel(limb_name)
```

Comment:
I don't know if other types exist, but this will also apply to other blunt weapons that guaranteed decap on hit. Just thought I'd say it

Reply:
The only other cause of blunt droplimb is wirecutter / power jaw suicide act, which should set them to DNR anyway

Reply:
👍 Thanks for looking into it